src/bin/ion/commands/beta/inspect.rs

```diff
@@ -1,22 +1,21 @@
-use std::cell::RefCell;
 use std::cmp::min;
 use std::fmt::{Display, Write};
 use std::fs::File;
 use std::io;
 use std::io::BufWriter;
 use std::ops::Range;
-use std::rc::Rc;
 use std::str::{from_utf8_unchecked, FromStr};
 
 use anyhow::{bail, Context, Result};
 use clap::{App, Arg, ArgMatches};
 use colored::Colorize;
-use ion_rs::{BinaryIonCursor, IonType, Reader, SymbolTable, SystemEventHandler};
 use ion_rs::result::IonResult;
 use ion_rs::text::writer::TextWriter;
+use ion_rs::{IonType, RawBinaryReader, SystemReader, SystemStreamItem};
 use memmap::MmapOptions;
 
-const ABOUT: &str = "Displays hex-encoded binary Ion alongside its equivalent text for human-friendly debugging.";
+const ABOUT: &str =
+    "Displays hex-encoded binary Ion alongside its equivalent text for human-friendly debugging.";
```

Comment:
Unfortunately, this diff contains whitespace changes introduced by a `cargo fmt` run that I forgot to isolate into its own commit. 🤦 Mea culpa. I'll add comments indicating the important logic changes.